Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/AggregationPriority/AggregationPriorityTest.cs

```diff
@@ -319,6 +319,7 @@ private async Task<IEnumerable<TaskRaw>> GetDistribution(int nRows)
   /// </summary>
   /// <param name="squareMatrixSize">The size of the square matrix.</param>
   [TestCase(20)]
+  [Ignore("Too big")]
```

Comment:
Is it possible to write a faster test that validates the same behavior ? That would be the best alternative

Reply:
I guess by reducing the matrix size from 20 to like 5 or something, but this tests core / and the queue, not what ExtC# does. SO I'm not sure it's relevant, especially for CI/CD